package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "webchat",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "devDependencies": {
+    "babel-runtime": "^6.26.0",
+    "html-webpack-plugin": "^5.0.0",
+    "ts-loader": "^8.0.16",
+    "typescript": "^4.1.3",
+    "webpack": "^5.21.2",
+    "webpack-cli": "^4.5.0",
+    "webpack-dev-server": "^3.11.2"
+  },
+  "dependencies": {
+    "@twilio/flex-webchat-ui": "^2.7.1"
+  }
+}
```

Comment:
This is a great starting point! It helped me with my typescript investigation in HRM!
One question: with this config, does webchat supports hot reloading? That means, if you modify any .ts file is the application compiling the .ts and reloading the app again?

Reply:
yes, more specifically the dependency `webpack-dev-server` is the one responsible for supporting hot reloading, in conjunction with some `webpack.config.js` configuration:
```
devServer: {
    contentBase: './build',
    compress: true,
    port: 9000,
  },
```